tutor/commands/upgrade/compose.py

```diff
@@ -164,6 +164,10 @@ def upgrade_from_quince(context: click.Context, config: Config) -> None:
     upgrade_mongodb(context, config, "5.0.26", "5.0")
     upgrade_mongodb(context, config, "6.0.14", "6.0")
     upgrade_mongodb(context, config, "7.0.7", "7.0")
+    fmt.echo_alert(
```

Comment:
nit: this would be moved to upgrade from sumac section because we are targeting this for nightly and makes sense to have this there.

Reply:
You mean to upgrade *to* Sumac, right? I agree, but we don't have an `upgrade_from_redwood` command yet, so we can make that change later in the sumac branch.

Reply:
Oh yeah, right. I meant upgrade_from_redwood section of sumac branch. I will take care of that in sumac branch once this branch is merged.

Reply:
I have updated this in sumac branch https://github.com/overhangio/tutor/pull/1139/commits/2ad4be04f55c6988ff8f0b01c30ac1d68a0ab595